### PR TITLE
filterChannelsByFrequencyRange fix

### PR DESCRIPTION
**Pull request by arhannan**
Modified logic to use intersection

### CODE REVIEW

File: src/harness/util.py [hunk 1]
```diff
@@ -405,9 +405,9 @@ def filterChannelsByFrequencyRange(channels, freq_range):
   return [
       channel for channel in channels
       if
-      channel['frequencyRange']['lowFrequency'] <= freq_range['highFrequency'] 
-      and
-      channel['frequencyRange']['highFrequency'] >= freq_range['lowFrequency']
+      (bool(set(range(channel['frequencyRange']['lowFrequency'],
```
Comment (kate-harrison) [inline src/harness/util.py]: Can you explain what problem you're trying to solve? As far as I can tell, the logic is the same (confirmed if all the unit tests still pass) but the new code is much less efficient.

Comment (arhannan) [reply src/harness/util.py]: Yes it is less efficient. See @gilschieber's comment [here](https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/pull/655#pullrequestreview-121118827)

Comment (kate-harrison) [reply src/harness/util.py]: Good point, I agree with his comment. But if you change the ">=" on line 410 to be just ">" -- and make no other changes -- I think it resolves this problem and the problem reported in #654. Do you agree?

**Comment by arhannan** [comment]
Thanks James and Kate. Updated

**Pull request merged**